app/index.js

```diff
@@ -311,6 +311,12 @@ JhipsterGenerator.prototype.askFor = function askFor() {
             name: 'enableTranslation',
             message: '(14/' + questions + ') Would you like to enable translation support with Angular Translate?',
             default: true
+        },
```

Comment:
I wouldn't put this in the end, but after the security question, so we have all the authentication questions at the same place.
Besides, does this work with the 3 different authent mechanisms?